codes/fermions/majorana_stab.yml

```diff
@@ -0,0 +1,51 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: majorana_stab
+physical: fermion
+logical: fermion
+
+name: 'Majorana stabilizer code'
+
+introduced: '\cite{arXiv:1703.00459}'
+
+description: 'Majorana fermion stabilizer codes are stabilizer codes whose stabilizers are products of an even number of Majorana fermion operators, analogous to Pauli strings for a traditional stabilizer code and referred to as ‘’Majorana stabilizers.’’ The codespace is the mutual \(+1\) eigenspace of all of the Majorana stabilizers. In such systems, Majorana fermions may either be considered individually or paired into ‘’complex fermions.’’ ’ 
+
+protection: 'Protects against de-phasing errors caused by coupling of fermion density to the environment and bit-flip errors caused by quasiparticle poisoning processes.'
+
+features:
+
+  # rate:
+
+  # encoders: 
+    
+  # transversal_gates:
+
+  # general_gates:
+    
+  # decoders: 
+   
+  # fault_tolerance:
+
+  # threshold: 
+
+# realizations:
+
+# notes:
+
+relations:
+  parents:
+    - code_id: fermions
+      detail: ‘The Majorana stabilizer code is a fermionic code.’
+    - code_id: stabilizer
+      detail: ‘The Majorana stabilizer code is a stabilizer code whose stabilizers are composed of Majorana fermion operators.’
+
+  cousins:
+    - code_id: weakly_self_dual
+      detail: ‘Classical weakly self-dual codes can be used to construct Majorana stabilizer codes. The direct relationship between the two codes follows from expressing the  Majorana strings as binary vectors – akin to the binary symplectic representation – and observing that the binary stabilizer matrix \(S\) for such a Majorana stabilizer code satisfies \(S\cdot S^T=0\) because it has commuting stabilizers, which is precisely the condition \(G\cdot G^T=0\) on the generator matrix \(G\) of a weakly self-dual classical code. A weakly self-dual classical code \(\mahcal{C}\) with parameters \([2N,k,d]\) yields a Majorana stabilizer code with parameters \([[N,N-k,d^\perp]]_\mathbf{f}\), where \(d^\perp\) is the code distance of the dual code \(\mathcal{C}^\perp\).’
+    - code_id: css
```

Comment:
- code_id: cyclic
   detail: 'Cyclic codes can be used to construct translation-invariant Majorana stabilizer codes, provided that they are also weakly self-dual.'